### DISCUSSION TITLE
What is the smallest possible Dogecoin transaction?

### DISCUSSION THREAD

**Discussion by viktorbunin** [dogecoin/dogecoin#2505] [Q&A] [answered]
Hi folks, My name is Viktor Bunin and I work at Bison Trails, a blockchain infrastructure provider acquired by Coinbase.

I'm trying to figure out the smallest transaction possible on Dogecoin. On Bitcoin the dust limit is 546 sats - anything smaller is rejected by the network. Is there any chance anyone knows what the dust limit is on Dogecoin?

Thank you very much, I greatly appreciate your help.

**Comment by ReverseControl** [comment]
See:

https://github.com/dogecoin/dogecoin/blob/master/src/policy/policy.h#L54

but, in 1.14.5 this will be reduced by a factor of ``100x``. Also, see:

https://github.com/dogecoin/dogecoin/blob/master/src/policy/policy.h#L48

**Comment by AbcSxyZ** [comment, marked as answer]
To add extra info to @ReverseControl comment :

There is currently change in fee policy, who started at the 1.14.4 and will be fully released at 1.14.5.
It will impact dust limit. You can see the discussion about the ongoing change in #2347 with the implementation in #2485.

They are lowering the current dust limit by x100 :  https://github.com/dogecoin/dogecoin/blob/e12d3f2a1ec9502de6675b490e78f85fa7b860a0/src/policy/policy.h#L54

If I'm right :
```
COIN = 100'000'000 #with 1 dogecoin = 100,000,000 koinu
DEFAULT_DUST_LIMIT = COIN / 100 = 100'000'000 / 100 = 1'000'000 koinu = 0.01 dogecoin
```
The current dust limit [the smallest possible transaction] is 1 Dogecoin, and will be lowered to 0.01 dogecoin in the next release (1.14.5).
